13output.tex

```diff
@@ -65,19 +65,19 @@
 
 \hypertarget{DerivedQuant}{}
 \subsection[Derived Quantities]{\protect\hyperlink{DerivedQuant}{Derived Quantities}}
 Before listing the derived quantities reported to the standard deviation report, there are a couple of topics that deserve further explanation.
 
 \hypertarget{VirginUnfished}{}
-\subsubsection[Virgin Spawning Biomass vs Unfished Spawning Biomass]{\protect\hyperlink{}{Virgin Spawning Biomass vs Unfished Spawning Biomass}}
-Unfished is the condition for which reference points (benchmark) are calculated. Virgin Spawning Biomass (B0) is the initial condition on which the start of the time-series depends. If biology or spawner-recruitment parameters are time-varying, then the benchmark year input in the forecast file tells the model which years to average in order to calculate ``unfished''. In this case, virgin recruitment and/or the virgin spawning biomass will differ from their unfished counterparts. Virgin recruitment and spawning biomass are reported in the mgmt\_quant portion of the sd\_report and are now labeled as ``unfished'' for clarity. Note that if ln(R0) is time-varying, then this will cause unfished to differ from virgin. However, if regime shift parameter is time-varying, then unfished will remain the same as virgin because the regime shift is treated as a temporary offset from virgin. Virgin spawning biomass is denoted as SPB\_virgin and spawning biomass unfished is denoted as SPB\_unf in the report file.
+\subsubsection[Virgin Spawning Biomass vs. Unfished Spawning Biomass]{\protect\hyperlink{VirginUnfished}{Virgin Spawning Biomass vs. Unfished Spawning Biomass}}
+Unfished is the condition for which reference points (benchmark) are calculated. Virgin Spawning Biomass (B0, Bzero) is the initial condition on which the start of the time series depends. If biology or spawner-recruitment parameters are time-varying, then the benchmark year input in the forecast file tells the model which years to average in order to calculate ``unfished''. In this case, virgin recruitment and/or the virgin spawning biomass will differ from their unfished counterparts. Virgin recruitment and spawning biomass are reported in the mgmt\_quant portion of the sd\_report and are now labeled as ``unfished'' for clarity. Note that if ln(R0) is time-varying, then this will cause unfished to differ from virgin. However, if regime shift parameter is time-varying, then unfished will remain the same as virgin because the regime shift is treated as a temporary offset from virgin. Virgin spawning biomass is denoted as SPB\_virgin and spawning biomass unfished is denoted as SPB\_unf in the report file.
```

Comment:
Why use "B0, Bzero"? I would think that it should be "$B_{0}$".

Reply:
I'm confused about B0 because it seems like it refers to Virgin Spawning Biomass and Unfished Spawning Biomass in the user manual, is that correct or are they different? I also see these referred to as SSB\_unf and SSB\_virgin in 7forecast.tex (lines 379, 388, and 390) which I assume is how it is in the SS3 output. @chantelwetzel-noaa looping you in here since you had a comment on line 485 of this file about Virgin Spawning Biomass.

Reply:
I would like to standardize all of this but also want to make sure it's reflective of what is shown in the ss3 output.